cloudinit/sources/helpers/vmware/imc/guestcust_util.py

```diff
@@ -520,6 +531,14 @@ def do_pre_custom_script(cust_cfg, custom_script, cust_cfg_dir):
             cust_cfg,
         )
         return False
+    except Exception as e:
```

Comment:
Since this is a copy of the `except` block above and `CustomScriptNotFound` is a subclass of exception, we should be just able to replace `CustomScriptNotFound` with `Exception`.

Reply:
Thanks, code updated with only replacing `CustomScriptNotFound` with `Exception`